### PR TITLE
BZ#1420807: Added instructions for launching jobs via 'oc run'

### PR DESCRIPTION
**Pull request by ddomingorh**
...also includes Known Limitation for restartPolicy; namely, that
'Never' and 'OnFailure' are effectively the same from the user
POV.

Content has already been reviewed by SMEs. Similar tests conducted on mini-lab.

This PR is similar to https://github.com/openshift/openshift-docs/pull/4651, but specific to 3.4. In 3.4, cron jobs are referred to as 'scheduled jobs', and the specs are similar as well.

### CODE REVIEW

**Comment by ddomingorh** [comment]
Corrected target branch... s/master/enterprise-3.4/.

Also, @soltysh @sferich888 @vikram-redhat @gaurav-nelson: here's the 3.4 version of https://github.com/openshift/openshift-docs/pull/4651

File: dev_guide/scheduled_jobs.adoc [hunk 1]
```diff
@@ -63,17 +62,83 @@ spec:
...
             image: perl
             command: ["perl",  "-Mbignum=bpi", "-wle", "print bpi(2000)"]
-          restartPolicy: Never
+          restartPolicy: OnFailure <4>
 ----
-
 1. Schedule for the job. In this example, a job will run every minute.
 2. Job template. This is similar to the xref:../dev_guide/jobs.adoc#creating-a-job[job example].
-====
+3. Sets a label for jobs spawned by this cron job.
```
Comment (soltysh) [inline dev_guide/scheduled_jobs.adoc]: scheduled job

**Review by soltysh — APPROVED** [review]
One small nit, otherwise lgtm

File: dev_guide/scheduled_jobs.adoc (hunk 1 above)
Comment (ddomingorh) [inline dev_guide/scheduled_jobs.adoc]: Fixed in latest. Thanks!

**Comment by vikram-redhat** [comment]
As discussed with @ddomingorh there were some issues with this branch. I am going to close this in favor of a new one.

**Comment by vikram-redhat** [comment]
Closed in favor of: https://github.com/openshift/openshift-docs/pull/4693

**Pull request closed without merging**